components/builder-depot/src/server.rs

```diff
@@ -869,6 +879,13 @@ fn upload_package(req: &mut Request) -> IronResult<Response> {
 
     let filename = depot.archive_path(&ident, &target_from_artifact);
 
+    let trident = ident.to_owned().into();
```

Comment:
Nitpick - consider rename to something like temp_ident instead of 'trident' ? :)